contracts/strategies/modules/voting/IVotingModule.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IStrategy} from "../../IStrategy.sol";
+
+interface IVotingModule is IStrategy {
+    function initializeVotingModule(bytes _data) external;
+    // initialize the voting itself, which could contain things like an ERC20 used to determine vote count
+
+    function vote(bytes memory _data, address _sender) external payable;
+    // called via allo.sol by users to allocate votes to a recipient
+    // this will update some data in this contract to store votes, etc.
+
+    function getResults(address[] memory _recipientId, bytes memory _data)
```

Comment:
@zobront is passing the results as bytes too flexible? I'm wondering if we should be requiring voting to always produce a fixed point number.

Reply:
@nfrgosselin It passes the results back as `ResultSummary[] memory`, which is address & number.

The extra data is for the input, which would be for example useful if we were needing to validate against a merkle root. It'll usually be empty.

Reply:
got it, thanks for clarifying!